docs/integrations/dell.md

```diff
@@ -0,0 +1,132 @@
+
+| Name                     | Description             |
+| ------------------------ | ----------------------- |
+| ecs.s3.endpoint          | ECS S3 service endpoint |
+| ecs.s3.access-key-id     | ECS Username            |
+| ecs.s3.secret-access-key | S3 Secret Key           |
+
+An ECS catalog requires that you configure a warehouse location where all data and metadata will be created.
+
+| Example                    | Description                                                     |
+| -------------------------- | --------------------------------------------------------------- |
+| ecs://bucket-a             | Use the whole bucket as the data                                |
+| ecs://bucket-a/namespace-a | Use a prefix to access the data only in this specific namespace |
+
+When you provide the `warehouse`, the last `/` will be ignored. The `ecs://bucket-a` is same with `ecs://bucket-a/`.
+
+### Runtime dependencies
+
+The Iceberg `runtime` jar supports different versions of Spark and Flink. If the version was not matched in the example, please check the related document of Spark and Flink.
+
+Even though the [Dell ECS client](https://github.com/EMCECS/ecs-object-client-java) jar is backward compatible, Dell EMC still recommends using the latest version of the client.
+
+### Spark
+
+For example, to use the Dell ECS catalog with Spark 3.2.1, you should create a Spark session like:
+
+```python
+from pyspark.sql import SparkSession
+
+spark = SparkSession.builder
+    .config("spark.jars.packages", "org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:0.14.0,com.emc.ecs:object-client-bundle:3.3.2")
+    .config("spark.sql.extensions", "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions")
+    .config("spark.sql.catalog.my_catalog", "org.apache.iceberg.spark.SparkCatalog")
+    .config("spark.sql.catalog.my_catalog.warehouse", "ecs://bucket-a/namespace-a")
+    .config("spark.sql.catalog.my_catalog.catalog-impl", "org.apache.iceberg.dell.ecs.EcsCatalog")
+    .config("spark.sql.catalog.my_catalog.ecs.s3.endpoint", "http://10.x.x.x:9020")
+    .config("spark.sql.catalog.my_catalog.ecs.s3.access-key-id", "<Your-ecs-s3-access-key>")
+    .config("spark.sql.catalog.my_catalog.ecs.s3.secret-access-key", "<Your-ecs-s3-secret-access-key>")
+    .getOrCreate()
+```
+
+Then, use `my_catalog` to access the data in ECS. You can use `SHOW NAMESPACES IN my_catalog` and `SHOW TABLES IN my_catalog` to fetch the namespaces and tables of the catalog.
+
+The related problems of catalog usage:
+
+1. The [pyspark.sql.SparkSession.catalog](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.sql.SparkSession.catalog.html#pyspark.sql.SparkSession.catalog) won't access the 3rd-party catalog of Spark, so please use DDL SQL to list all tables and namespaces.
+
+
+### Flink
+
+Use the Dell ECS catalog with Flink, you first must create a Flink environment.
+
+```python
+import requests
```

Comment:
Is it good enough to provide a python script to download the flink jars?  I see the script will need the `pyflink` and `requests`  dependencies and we don't provide any more details about how to run the python script. I'm afraid most of people won't be able to run this python script successfully in their hosts.

I think it's simple enough to wget the flink jars in two line bash scripts and show the following SQL Execution by using flink SQL.

Reply:
Now the scripts are bash-script.